SingularityUI/app/actions/api/user.es6

```diff
@@ -0,0 +1,41 @@
+import { buildApiAction, buildJsonApiAction } from './base';
+
+export const FetchUser = buildApiAction(
+  'FETCH_USER',
+  {url: '/auth/user'}
+);
```

Comment:
Can we move this back to `auth.es6`? We're trying to match the actions folder structure with the API paths.